pages/give-devices.js

```diff
@@ -9,21 +9,19 @@ import SelectFormControl from '../components/form/SelectFormControl';
 import TextareaFormControl from '../components/form/TextareaFormControl';
 
 // Use ISO 3166 country codes per https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
-const COUNTRIES = [
-  { value: 'US', label: 'United States' },
-  { value: 'CA', label: 'Canada' },
-  { value: 'MX', label: 'Mexico' },
-  { value: 'DE', label: 'Germany' },
-  { value: 'LV', label: 'Latvia' },
-  { value: 'IN', label: 'India' },
-  { value: 'BD', label: 'Bangladesh' },
-  { value: 'PH', label: 'Philippines' },
-];
+const countryList = require('country-list');
```

Comment:
The `country-list` package appears to be taking data from [this csv file](https://github.com/datasets/country-list/edit/master/data.csv) which we could easily download and convert to a json file for our use case. This would prevent us from having to add yet another dependency. ((The data is actually converted to json for this package at `node_modules/country-list/data.json`))

Extracting the country names from this data would also allow us to easily modify the text such as where notes have been added i.e `(french part)`